sections/sentinels.tex

```diff
@@ -0,0 +1,185 @@
+    \begin{itemize}
+        \item First die: --1 west, +1 east, 0 reroll
+        \item Second die: --1 north, +1 south, 0 reroll
+    \end{itemize}
+    \item For more equal distribution, in rounds 7 and 9 the enemy spawns on the opposite side
+    than in the round before. Rounds 8 and 10 are all random.
+    \item If the tile to spawn an enemy on isn't discovered yet, flip it over and orient as preferred.
+    \item If there is a hero on the field where the enemy spawns, a fight is initiated immediately.
+    \item Enemy armies move at the end of every turn, starting with the same turn they spawned.
+    \item Enemy armies have 3 Movement Points per round.
+    \item Enemy armies move as described in the rulebook (p. 33) but instead of capturing they destroy
+    everything in their way. Place black cubes on any field they pass and treat them as empty from that point.
+    \item If an enemy's movement could go two different ways the players decide which way to go.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Combat with enemy armies}}
+
+\begin{itemize}
+    \item During a battle with an enemy army a hero can retreat every time a friendly unit is about to 
+    activate. Retreating main hero loses all remaining Movement Points and is returned to an
+    allied town or settlement of their choice, keeping all remaining units.
+    \item Retreating secondary heroes are removed from the game, remaining units are kept. 
+    \item Killed enemy units do not respawn. 
+\end{itemize}
+
+\subsection*{\MakeUppercase{Boss army}}
+
+\begin{itemize}
+    \item On Round 10 a Boss Army spawns and is different from previous armies, because it gets reinforced 
+    from a fixed pool of neutral units (reinforcement pool = number in brackets in the table below).
+    \item At the beginning of the 2$^{nd}$ combat round and every following round 2 units reinforce the
+    remaining army from the shuffled reinforcement pool (up to the maximum of 5 units) to
+    continue the fight until either the player retreats or every neutral unit from the reinforcement
+    pool is killed. \textit{If the number of enemy units at the beginning of a round is 1 or lower, draw up to 
+    a total of 4 units (e.i. there are always 4 or 5 enemy units on the board after reinforcing, if available).}
+    \item Place the reinforcing units on the enemy base line starting from the left with the lowest initiative
+    (\svg{unit_ranged} units first). If necessary continue on the next line (in the same order).
+    \item When a player retreats the enemy army draws up to the "minimum" of 4 units and takes these
+    as starting units to the next fight. 
+\end{itemize}
+
+\hommtable[]{18}{
+  \centering
+  \medskip
+  \textbf{Strength of Enemy Armies}\\
+  \bigskip
+
+  \newcommand{\bronze}[0]{\svg[12]{bronze}}
+  \newcommand{\silver}[0]{\svg[12]{silver}}
+  \newcommand{\golden}[0]{\svg[12]{golden}}
+  \newcommand{\azure}[0]{\svg[12]{azure}}
+
+  \phantom{\bronze\silver\golden\azure}
```

Comment:
Is there a better way to work with svg than this? I need to use them before `tabularx` or it doesn't work